docs/kqueen.rst

```diff
@@ -428,6 +428,32 @@ To manually add an existing Kubernetes cluster:
 As a result, the Kubernetes cluster will be attached in a read-only mode.
 
 
+Kqueen UI Cluster actions:
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+* Currently supported cluster operations:
```

Comment:
operations are: delete, create and so on, here you are describing cluster properties

Reply:
agree, but prefer to wait mzlatkova review too,